### PR TITLE
Roll version numbers for 1.9.4 release

### PR DESCRIPTION
**Pull request by baylesj**

### CODE REVIEW

File: include/json/version.h [hunk 1]
```diff
@@ -9,7 +9,7 @@
 // 3. /CMakeLists.txt
 // IMPORTANT: also update the SOVERSION!!
 
-#define JSONCPP_VERSION_STRING "1.9.3"
+#define JSONCPP_VERSION_STRING "1.9.4"
 #define JSONCPP_VERSION_MAJOR 1
 #define JSONCPP_VERSION_MINOR 9
 #define JSONCPP_VERSION_PATCH 3
```
Comment (utkarshayachit) [inline include/json/version.h:15]: updating the PATCH here was missed.

Comment (cdunn2001) [reply include/json/version.h:15]: Also

     #define JSONCPP_VERSION_PATCH 3

**Comment by cdunn2001** [comment]
This was merged before **AppVeyor** had passed. Not good!

I'm trying to figure out when **TravisCI** stopped running. We're lucky **AppVeyor** still works I think.

**Comment by cdunn2001** [comment]
Weird. My latest PR failed in TravisCI -- so obviously it *did* run in TravisCI -- but there is no TravisCI notification in GitHub.

Possibly related: https://travis-ci.community/t/github-ci-notifiactions-from-travis-stopped-working/9238/3

Still investigating ...

**Comment by cdunn2001** [comment]
Fixed it by adding a new "Branch Protection Rule". No worries.

**Pull request merged**